lib/paypalhttp/serializer/json.js

```diff
@@ -10,7 +10,7 @@ class Json {
   }
 
   contentType() {
-    return /^application\/json/;
+    return /^application\/(json|JSON)/;
```

Comment:
why not just make the regex case insensitive?
```
return /^application\/json/i;
```

Reply:
you're right it could be better to do so